### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by krisbobo**
## Portfolio-setup and mobile version: Contact form ##
In this section, we:
- Added a contact form which accepts a name(which accepts up to 30 characters), email and message field(which accepts up to 500 characters) as input
- Ensured that the input fields only accept its suitable value
- Ensured that a user cannot submit the form if one or more of its fields are missing
- Allowed for data collection by using the [Formspree Service](https://formspree.io/html/)

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -343,3 +343,64 @@ p {
...
+
+.contact-button {
+  padding: 15px;
+  color: white;
+  border: none;
+  margin-top: 10px;
+  margin-left: 35%;
+  background-color: #36b37f;
+  border-radius: 4px;
+}
```
Comment (BenMukebo) [inline css/style.css]: - [x] Kindly make sure to sticks to the design [templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) in Figma as much as possible, this includes the font, colours, border margin, padding.... The `Get in Touch` button doesn't really match the original design very much yet, please use the same value from the figma templete.
- eg: padding: 12px 12px 12px 12px;
- font-weight : 500;
- .......


||      Your       ||        Figma       ||
|-|----------------------|-| ----------------------|-|
||badbtn ||btnfigma||

File: index.html [hunk 2]
```diff
@@ -189,6 +189,27 @@ <h4 class="lang-text">Skills</h4>
 
 </div>
 </section>
+<footer>
+  <p class="contact-description">I'm always interested in hearing about new projects, 
+    so if you'd like to chat please get in touch.</p>
+  <form class="form" action="https://formspree.io/f/xpzbonqw" method="post">
+    <ul>
+      <li><input class="form-input" type="text" placeholder="Full name" maxlength="30" required></li>
```
Comment (BenMukebo) [inline index.html:197]: - [x] According to the project requirements, you need to use a CSS Flexbox property to layout the form elements, please add it.

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi Team,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- Descriptive PR. ❤️ 
- GitHub flow was correctly used.  ✔️  
- Descriptive README file. ✔️  
- The form is submitted successfully with valid input values. ✔️  
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


- [OPTIONAL ] Please let's make sure that contribution to a group project is distributed equally among the members( commit messages should be fairly equal).

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**